modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ResourceBundleLoaderProviderUtil.java

```diff
@@ -37,17 +34,16 @@ public ResourceBundleLoader getResourceBundleLoader(
 			ResourceBundleLoaderUtil.getPortalResourceBundleLoader());
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
-		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
-			bundleContext, ResourceBundleLoader.class, "bundle.symbolic.name");
-	}
+	private static final ServiceTrackerMap<String, ResourceBundleLoader>
```

Comment:
Hey @Valmir-unicap, you do not need this tracker. Use the `ResourceBundleLoaderUtil` instead. Thx